interface/patient_file/summary/disclosure_full.php

```diff
@@ -55,15 +55,18 @@
 ?>
 <html>
 <head>
+<link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap-3-3-4/dist/css/bootstrap.min.css">
+<?php if ($_SESSION['language_direction'] == 'rtl') { ?>
+    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/bootstrap-rtl-3-3-4/dist/css/bootstrap-rtl.min.css">
+<?php } ?>
```

Comment:
nice on above including the rtl bootstrap css. Also rec doing this in code above this (3 places in code above where you include bootstrap css without the rtl bootstrap css).